features/dependencies.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Install dependencies Tuist.
```

Comment:
Since we don't want the `Tuist/Dependencies` directory to be added to the Git repository by default, I'd recommend extending the git ignore for the `app_with_framework_and_tests_and_dependencies` fixture, to ignore the `Tusit/Dependencies` directory. Moreover, I'd update the gitignore of the template that we use to scaffold new projects to include `Tuist/Dependencies`. You can add a new section [below this one](https://github.com/tuist/tuist/blob/main/Templates/Gitignore.stencil#L65):

```
### Tuist managed dependencies ###
Tuist/Dependencies
```

Reply:
1. I have already added `Tuist/Dependencies` to `.gitignore` for `app_with_framework_and_tests_and_dependencies`. You can check it [here](https://github.com/tuist/tuist/blob/b8fb0bc95d46bf34a59c9cc1c605967afdf67a50/fixtures/app_with_framework_and_tests_and_dependencies/.gitignore#L66). Of course I can change `### Generated ###` title to `### Tuist managed dependencies ###`.
2. I am not sure about update the `gitignore` of the template. It should be user decision if he/she want to push dependencies to git.

Reply:
> I am not sure about update the gitignore of the template. It should be user decision if he/she want to push dependencies to git.
 
I'm inclined git ignoring them by default. If they disagree with Tuist's stance, they can delete the line from the `.gitignore` file.